arch/arc/include/swap_macros.h

```diff
@@ -104,7 +125,7 @@
 #ifdef CONFIG_FPU_SHARING
 	ld_s r13, [r2, ___thread_base_t_user_options_OFFSET]
 	/* K_FP_REGS is bit 1 */
-	bbit0 r13, 1, 2f
+	bbit0 r13, 1, 3f
```

Comment:
Let's use labels here instead of `2f`, `3f`, `4f, here. It's getting really cryptic with such level of nesting.

Reply:
fiexed